setup/windows/README.md

```diff
@@ -49,6 +49,6 @@ $ ansible-playbook -i ../ansible-inventory ansible-playbook.yaml -vv
 ```
 
 The servers should logon automatically at boot and start the Jenkins slave, but there is no automatic restart procedure for crashes (for now).
-Hence, to start Jenkins after running `ansible-playbook`, the server must be restarted or the slave must be manually started with RDP.
```

Comment:
Is there no other way on windows to update the PATH than restarting the machine?

Presumably you only have to restart the server the first time you run the script, as if you run it again to update the machines later `curl` and `java` will already be in the path.

Also, does Ansible not have a way to restart the server once it's finished (@joaocgreis @piccoloaiutante ) ?